http-servers/html-server.js

```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+const fs = require('fs');
+
+
+const server =  http.createServer((request, response) => {
+    response.writeHead(200, {'Content-Type': 'text/html'});
+    fs.createReadStream(`${process.cwd()}/http-servers/index.html`).pipe(response);
```

Comment:
It is better to use `join(__dirname, PATH_TO_FILE)` instead of string path, due to cross-platform operability. And move path string to the constant variable.

Reply:
Resolved